packages/blade/src/components/Toast/_decisions/decisions.md

```diff
@@ -0,0 +1,267 @@
+# Toast 🏷️
+
+Toasts are used to provide feedback to the user after an important action has been performed. 
+Toasts can also be used to provide feedback to the user when a system event occurs, such as when a file is saved.
+
+<img width="426" alt="image" src="./toast-thumbnail.png" />
+
+## Design
+
+- [Toast - Figma Design](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade-DSL?type=design&node-id=7665-27414&mode=design&t=UNInCMmP1iFCu9je-0)
+
+## Features
+
+- Stackable
+- Auto dismissable / manual dismissable
+- `informational` or `promotional` types
+
+## Anatomy
+
+### Informational Toast
+
+<img src="./toast-anatomy.png" width="100%" alt="Toast anatomy" />
+
+### Promotional Toast
+
+<img src="./example-promotional-toast.png" width="380" alt="Promotional toast example" />
```

Comment:
> add anatomy wala screenshot here
Design hasn't created anatomy illustration for promo toast. // @saurav12 

> primarily because icon is asset in promotional
There is no difference of anatomy between info/promo toast. Both are 1:1 

- Leading: icon or assset
- Content: JSX element (to put bold/italic text inside info)
- Action Button
- Dismiss Button